docs/howto/tracing.md

```diff
@@ -113,16 +154,11 @@ class Workload:
             ...
 ```
 
-## Adding tracing to charm libraries
+Refer to the {ref}`ops_tracing` reference for the canonical usage example, configuration
+options, and API details.
 
-- At the top of your charm library, `import opentelemetry.trace`.
-- After the imports in your charm library, create the tracer object as
-  `tracer = opentelemetry.trace.get_tracer(name)` where the name could be your
-  charm library name, or Python module `__name__`.
-- See the [Custom spans and events](custom-spans-and-events) section above to
-  create OpenTelemetry spans and events in the key places in your charm library.
 
-## Migrating from the charm\_tracing charm library
+### Replace the charm_tracing library
```

Comment:
My feeling is that people would search for "migrating" (or "migrate", and hopefully the search has some stemming). Could we stick with that word?